vendor/arbiter/arbiter.cpp

```diff
@@ -4601,6 +4601,10 @@ std::string encodeAsHex(const std::string& input)
 #include <arbiter/util/types.hpp>
 #endif
 
+#ifdef WINDOWS
```

Comment:
This code is part the arbiter repo:
https://github.com/connormanning/arbiter
Couple of problems with modifying our version:
- we will have to maintain a fork
- I don't thiink you solve the root of the problem
I think you should submit a PR to the arbiter tepo first, get feedback from the maintainer with the right fix and then we can integrate here. @smileprem any objection?

Reply:
That's the right approach. Let's do that. However, should we wait for the PR to be accepted to move on with the current code changes?.

Can we fork the aribiter, make changes and submit a PR, while that is in process, use our arbiter codebase for `helix-master` branch?. I know the code will not be merged to master until then, but we can finish the refactoring and complete the testing?.

I am trying to save some time here. Finally there will be a merge from `helix-master` to `master` anyway.

Reply:
@pravinshinde825 could you please submit a PR in arbiter in order to start the conversation? I'll merge this straight after. Regarding the cmake seetings above, could you please raise an issue on PDAL repo and ask what's up? It is supposed too be cross platfrom and they use MSVC 2017. SO no eason it does not work for us. It is really important we keep close to the original repos, they are moving quite fast. Thx!

Reply:
@nicolas-chaulet Sure I'll check arbiter repo and create PR there. But I think before creating PR we should raise an issue to arbiter repo. What do you think?